### ISSUE TITLE
xWSMan module available but needs community assistance

### ISSUE THREAD

**Issue by mgreenegit** [PowerShell/DscResources#215]
A module for WSMan is available but needs assistance from the community. Review, tests, and docs.
https://github.com/robwi/xWSMan

HELP WANTED

**Comment by mbreakey3** [comment]
We actually have a new module submission for WSManDsc open right now. Just waiting on review:
https://github.com/PowerShell/DscResources/issues/75

**Comment by mgreenegit** [comment]
Nice.  It also looks like there are some features that are unique across the two, so perhaps they could be compared and some capabilities could be merged.

**Comment by mbreakey3** [comment]
Yes, that would be great!

**Comment by PlagueHO** [comment]
@mgreenegit , @mbreakey3 - cool! I'll definitely help get that in line. Looks like the two resource modules will compliment each other really well.

This might be an opportunity to define a "merge" process for merging files from two repos that don't share a common ancestor. @mbreakey3 - have you guys ever done this and if so is there an existing process/method we could use? Or should this just be a case of "Copy/Paste" :grin:

**Comment by johlju** [comment]
@PlagueHO Try out this one, and see if it needs improving.
[How to continue working on a pull request when an author (contributor) is unable to complete it](https://github.com/PowerShell/DscResources/blob/master/GettingStartedWithGitHub.md#how-to-continue-working-on-a-pull-request-when-an-author-contributor-is-unable-to-complete-it)

**Comment by mbreakey3** [comment]
@PlagueHO, @johlju - Yeah, we don't have a defined 'merge' process for this situation yet. The easiest would be to get one merged in and then have the other contributor create a PR to add/update their own changes. However, since we've been a bit behind on reviewing new module submissions, if any of you want to work on merging the two together beforehand, that might make the process faster. That would require you to work in each other's repos though (whichever one you choose to be the 'base' module).

**Comment by PlagueHO** [comment]
@mbreakey3 - as the WSManDsc should be nearing being moved over (2 out of 3 reviews complete I think from memory) I'm not sure I'd want to muddy things up by adding to it till after it has been merged. 

Also, given that the repo is in that special review state (reviewBase/reviewTarget branches) then I'm not even sure of the pattern that would be adopted here. It could end up messing up the existing review - and I'm sure none of want to go through it again :grin:

**Comment by johlju** [comment]
@PlagueHO would you be okay if we move this issue to a new issue in the repo https://github.com/PlagueHO/WSManDsc? Maybe someone can work on it there instead?

**Comment by PlagueHO** [comment]
Fine with me...@mgreengit? I'd definitely like to incorporate @RobWi's resource into mine as it provides some very complimentary features. Just needs HQRM, tests & examples etc. But nothing too major.

**Comment by johlju** [comment]
This issue was moved to https://github.com/PlagueHO/WSManDsc/issues/18 since it has a higher chance that it can get help there. Closing this issue at this time.
